Update broadcast with alpha to reduce num of launching kernels.

xiaomengy opened the pull request:
Summary: Update broadcast with alpha to reduce num of launching kernels.

Differential Revision: D9175824

houseroad reviewed (approved):
Look good. Please make sure the CI passes.

xiaomengy committed:
Update broadcast with alpha to reduce num of launching kernels. (#10235)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/10235

Update broadcast with alpha to reduce num of launching kernels.

Differential Revision: D9175824

fbshipit-source-id: fdea28c3c834265c65af2fe0bfb758f14f442f48
